src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int[] salaries = new int[names.length];
+        LocalDate dateFromLocal = getLocalDateFromString(dateFrom);
+        LocalDate dateToLocal = getLocalDateFromString(dateTo);
+        for (String line: data) {
+            String[] parsedData = line.split(" ");
+            for (int i = 0; i < names.length; i++) {
```

Comment:
can we use for-each here as well?

Reply:
No, because i'll use his index "i" futher in line 19, otherwise, i should create this variable. First index of salaries array has salary of the first name in array names e.t.c